doc/Alerting/Hostdependencies.md

```diff
@@ -0,0 +1,7 @@
+source: Alerting/Hostdepencies.md
```

Comment:
This file won't get generated until you add it mkdocs.yaml. As it's a hidden one (part of another heading) then add it to hidden at the bottom and give it a good title.

Reply:
Is mkdocs.yaml in another repo? I couldn't find it.

Reply:
Nope, should be in the root of this one.